src/index.tsx

```diff
@@ -20,10 +20,17 @@ export function ReactAccordion(props: ReactAccordionProps) {
     <ul>
       {items.map((item, index) => (
         <li key={item.name}>
-          <details aria-label={item.name} open={item.open}>
-            <summary onClick={(event: SyntheticEvent) => typeof onClick === 'function' && onClick(event, index)}>{item.heading}</summary>
-            {item.child}
-          </details>
+          {item.child
```

Comment:
Worth splitting this out in to something like `LineItem`?

Reply:
Hmm not sure. Maybe overkill at the moment. But if it gets bigger, yeah. We should split out the onClick into a handleClick function though.